explorer/src/pages/Attestation/RelatedAttestations/index.tsx

```diff
@@ -1,9 +1,9 @@
 import { Attestation } from "@verax-attestation-registry/verax-sdk";
 import { EyeOffIcon } from "lucide-react";
+import { Link } from "react-router-dom";
```

Comment:
here we need to return everything as it was
`import { Link } from "@/components/Link";`